### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by JacobSaudan**
Hey!

### CODE REVIEW

**Review by wingardtw — APPROVED** [review]

File: authentication/forms.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#from laddr_site.models import Profile
+#from django.contrib.auth import get_user_model
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.contrib.auth.models import User
+
+
+#class UserForm():
```
Comment (wingardtw) [inline authentication/forms.py]: Remove code that is commented out

File: authentication/views.py [hunk 2]
```diff
@@ -33,7 +34,26 @@ def login_page(request):
...
+
+	if request.method =="POST":
+		user_form = SignupForm(data=request.POST)
+		if user_form.is_valid():
+			user_form.save()
+			raw_password = user_form.cleaned_data.get('password1')
+			username = user_form.cleaned_data.get('username')
+			user = authenticate(username=username, passowrd=raw_password)
+			login(request,user)
+			# Not currently working: return redirect(sign_up)
```
Comment (wingardtw) [inline authentication/views.py]: I thought we fixed this on skype

**Review by wingardtw — CHANGES_REQUESTED** [review]
See comments

File: authentication/views.py (hunk 2 above)
Comment (JacobSaudan) [reply authentication/views.py]: Whoops, forgot to push after making those changes

File: authentication/forms.py (hunk 1 above)
Comment (JacobSaudan) [reply authentication/forms.py]: 👌

**Pull request merged**